2025/10/23 13:45:52 [error] 242#242: *65 api could not be resolved (2: Server failure), client: 172.18.0.1, server: api.localhost, request: "GET / HTTP/1.1", host: "api.localhost", referrer: "http://localhost:81/"
2025/10/23 13:45:56 [error] 242#242: *65 api could not be resolved (3: Host not found), client: 172.18.0.1, server: api.localhost, request: "GET /favicon.ico HTTP/1.1", host: "api.localhost", referrer: "http://api.localhost/"
2025/10/23 14:32:33 [error] 244#244: *267 api could not be resolved (2: Server failure), client: 172.18.0.1, server: api.localhost, request: "GET / HTTP/1.1", host: "api.localhost", referrer: "http://localhost:81/"
2025/10/23 14:32:33 [error] 244#244: *267 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: api.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.3:8080/favicon.ico", host: "api.localhost", referrer: "http://api.localhost/"
2025/10/23 14:44:49 [error] 242#242: *367 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: api.localhost, request: "GET / HTTP/1.1", upstream: "http://172.18.0.3:8080/", host: "api.localhost", referrer: "http://localhost:81/"
2025/10/23 14:44:49 [error] 242#242: *367 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: api.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.3:8080/favicon.ico", host: "api.localhost", referrer: "http://api.localhost/"
2025/10/23 14:45:54 [error] 260#260: *379 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: phpmyadmin.localhost, request: "GET / HTTP/1.1", upstream: "http://172.18.0.6:8080/", host: "phpmyadmin.localhost", referrer: "http://localhost:81/"
2025/10/23 14:45:54 [error] 260#260: *379 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: phpmyadmin.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.6:8080/favicon.ico", host: "phpmyadmin.localhost", referrer: "http://phpmyadmin.localhost/"
2025/10/23 14:46:10 [error] 261#261: *382 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: phpmyadmin.localhost, request: "GET / HTTP/1.1", upstream: "http://172.18.0.6:8080/", host: "phpmyadmin.localhost", referrer: "http://localhost:81/"
